probe/cobra.go

```diff
@@ -0,0 +1,35 @@
+package probe
+
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+)
+
+func CobraCommand() *cobra.Command {
```

Comment:
I don't know if we `lint` istio/pkg but can you put a comment in front of every exported function so that we can generate docs?

Reply:
Done.

Reply:
Done.